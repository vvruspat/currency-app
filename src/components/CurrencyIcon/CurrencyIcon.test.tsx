import { render, screen } from "@testing-library/react";
import { CurrencyIcon } from ".";

test("renders CurrencyIcon", () => {
  render(<CurrencyIcon data-testid="test-currency-icon" currency="USD" />, {});

  const CurrencyIconElement = screen.getByTestId("test-currency-icon");

  expect(CurrencyIconElement).toBeInTheDocument();
  expect(CurrencyIconElement).toMatchSnapshot();
});
