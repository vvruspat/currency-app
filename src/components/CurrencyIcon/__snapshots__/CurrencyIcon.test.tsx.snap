// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CurrencyIcon 1`] = `
<div
  class="currency-icon currency-icon-USD"
  data-testid="test-currency-icon"
/>
`;
